src/vm/virtualcallstub.cpp

```diff
@@ -974,7 +974,7 @@ void VirtualCallStubManager::Reclaim()
 
 //----------------------------------------------------------------------------
 /* static */
-VirtualCallStubManager *VirtualCallStubManager::FindStubManager(PCODE stubAddress,  StubKind* wbStubKind)
+VirtualCallStubManager *VirtualCallStubManager::FindStubManager(PCODE stubAddress,  StubKind* wbStubKind, bool usePredictStubKind)
```

Comment:
A nit - for the sake of consistency, it would be better to use `BOOL` instead of `bool` to match other functions in this class.